docs/src/main/paradox/testing-routes.md

```diff
@@ -57,7 +57,7 @@ We simply construct a raw `HttpRequest` object and pass it into the route using
 After all, the route is a function of `HttpRequest => Future[HttpResponse]` - here we don't need to explicitly write code
 that's awaiting on the response, it's handled for us.
 
-Inside the check block we can inspect all kinds of attributes of the received response, like `status`, `contentType` and 
+Inside the check block we can inspect [all kinds of attributes](https://doc.akka.io/docs/akka-http/current/routing-dsl/testkit.html?language=scala#table-of-inspectors) of the received response, like `status`, `contentType` and 
```

Comment:
(we could use [extref](https://developer.lightbend.com/docs/paradox/latest/features/linking.html#extref-directive) here, but not essential

Reply:
Thanks for the indication! I will push another commit later.